packages/tocco-ui/src/EditableValue/utils.js

```diff
@@ -70,3 +70,17 @@ export const parseLocalePlaceholder = countryCode => {
 export const convertStringToNumber = stringValue => (
   !stringValue || isNaN(stringValue) ? null : parseFloat(stringValue)
 )
+
+/*
+ * Convert two numbers as hours and minutes to milliseconds
+ */
+export const calculateMilliseconds = (hours, minutes) => {
+  const hoursMilliseconds = (hours || 0) * 60 * 60000
```

Comment:
This method will break with negative values since -1 is false. does that matter?

Reply:
It shouldn't because Time and Duration components prevent negative inputs already.